### PR TITLE
copyedit profiling guide

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

**Comment by lpinca** [comment]
LGTM.

**Comment by stevemao** [comment]
Maybe leave a reference of where this is copied from?

File: locale/en/docs/guides/simple-profiling.md [hunk 1]
```diff
@@ -133,13 +133,13 @@ passwords, or inside the Express framework itself.
...
-form isolate-0x124353456789-v8.log. In order to make sense of this file, we need
-to use the tick processor included in the Node.js source at
-<nodejs_dir>/tools/v8-prof/tick-processor.js. It is important that the version
-of the tick-processor that you run comes from the same version of node source as
-version of node used to generate the isolate file. This will no longer be a
-concern when the tick processor is [installed by default][]. The raw tick output
-can be processed using this tool by running:
+form `isolate-0xnnnnnnnnnnnn-v8.log` (where `n` is a digit). In order to make
+sense of this file, we need to use the tick processor included in the Node.js
+source at `<nodejs_dir>/tools/v8-prof /tick-processor.js`. It is important that
```
Comment (fhemberger) [inline locale/en/docs/guides/simple-profiling.md:138]: Superfluous space in command

**Comment by fhemberger** [comment]
@stevemao It's the exact same text as before (with adjusted wrapping) and new "`isolate-0xnnnnnnnnnnnn-v8.log` (where `n` is a digit)".

**Comment by stevemao** [comment]
Oh from the commit message I thought it was copied from somewhere.

**Comment by Trott** [comment]
@stevemao I was using _copyedit_ in this context: https://en.wikipedia.org/wiki/Copy_editing 

I use that word a lot in my commit messages (when I'm fixing typos, punctuation, formatting, etc.) but if it's confusing to a lot of people, I can try to use a synonym instead.

**Comment by stevemao** [comment]
@Trott It's Ok I guess. My English is very bad :smile:

File: locale/en/docs/guides/simple-profiling.md [hunk 2]
```diff
@@ -133,13 +133,13 @@ passwords, or inside the Express framework itself.
...
+the version of the tick-processor comes from the source for the version of node
```
Comment (evanlucas) [inline locale/en/docs/guides/simple-profiling.md:139]: Hm, should we also mention the `--prof-process` flag now?

**Comment by fhemberger** [comment]
Merged via 6a61f69, thanks!

**Pull request closed without merging**